api/pkg/service/resource/resource.go

```diff
@@ -297,6 +297,12 @@ func initResource(r model.Resource) *resource.ResourceData {
 		RawURL:              getStringReplacer(lv.URL).Replace(lv.URL),
 		UpdatedAt:           lv.ModifiedAt.UTC().Format(time.RFC3339),
 	}
+
+	// If latest version is deprecated then add it only
+	if lv.Deprecated == "true" {
```

Comment:
Does this mean like if the resource is deprecated then this field will be there in response?

Reply:
yeah